README.md

```diff
@@ -164,7 +164,7 @@ color than the rest of the line, then you can add the following snippet to the
 `dotspacemacs/user-config` section in your `~/.spacemacs` file:
 
 ```elisp
-(setq ns-use-srgb-colorspace nil)
+(setq powerline-image-apple-rgb t)
```

Comment:
I think the following description will not be true if you used this variable as it will only affect colour generation of powerline separators and not all colorus of Emacs.